devtools/common/src/main/filtered/extensions.json

```diff
@@ -497,5 +497,21 @@
     ],
     "groupId": "io.quarkus",
     "artifactId": "quarkus-reactive-pg-client"
+  },
+  {
+    "name": "Apache Kafka Client",
+    "labels": [
+      "kafka"
+    ],
+    "groupId": "io.quarkus",
+    "artifactId": "quarkus-kafka-client"
+  },
+  {
+    "name": "Apache Kafka Streams",
+    "labels": [
+      "kafka"
```

Comment:
It's a label used for Search. So we should have kafka anyway (and we can have kafka-streams too)

Reply:
Yes, that's what I wanted to ask, too. What's the purpose of this label. If it is for searching suitable extensions, I could add both, "kafka" and "kafka-streams".

Reply:
`-Dextension=kafka` must work and add the MP Kafka support. That's why this one can't be kafka.

Reply:
@cescoffier well, you can discuss with @emmanuelbernard but that's definitely not the direction he wanted us to go.

Otherwise, we won't be able to add proper labels to any extensions.

Reply:
using `kafka` will give the list making the user confused... 
Kafka is one of the things where it should work without confusion.

Reply:
Hum, ok. Unique identification goes a bit against common interpretation of what a "label" is. Also that `-Dextension=kafka` adds MP Kafka instead of the plain Kafka client could come to the surprise of people. But if that's what you folks want here, then be it.

Reply:
Well. I clearly disagree with @cescoffier. Labels will be used for searching extensions in the UI and for all the others, we use labels that could create confusion. 

That’s what I fought to only use the artifact name (with some clever shortcut to avoid requiring quarkus- for instance). But @emmanuelbernard was against it. 

As long as we accepted this behavior, we must accept potential conflicts. I don’t like it but that’s how it is. 

That being said, I just saw he thought about the recent use case I reported with arc not being installable: https://github.com/quarkusio/quarkus/issues/2737 .

With what he suggests, we could keep kafka as a label. So I would say let’s add it and let’s refine our algorithm to avoid the issue!

Reply:
Then, it's a big breaking change as most of our quickstarts and demos are based on short names (called labels so far). Maybe we need to introduce a short name things too. 

About kafka not using the default kafka client, that's an opinionated choice we made.

Reply:
I laid down some thoughts around it https://github.com/quarkusio/quarkus/issues/2737
I don't think most were thinking that labels would be globally unique identifiers. I personally saw it as a way for the search to return all relevant things about $label
Now with my proposal to even remove labels from the `-Dextensions` algorithm, it would mean that MP Reactive Messaging would not show up (unless one does `-Dsearch`).

I do understand Clement's point to try and guide Kafka message consumers towards MP Reactive Messaging. But it kinda make sense for kafka streams to have a kafka label. If we assume that people don't know the difference between kafka streams and kafka client, then I think we are doomed on such a minimalist UI as the starter or the `add-extensions` command.